### PR TITLE
test(sdk-coin-coreum): add tests for wallet recovery

### PR DESCRIPTION
**Pull request by hitansh-madan**
Ticket: WIN-1076

- Adds tests for wallet recovery for coreum. Also tested with WRW changes locally [[PR](https://github.com/BitGo/wallet-recovery-wizard/pull/353)] : [tx(explorer)↗](https://explorer.testnet-1.coreum.dev/coreum/transactions/4543060FB87C2C3166BAE077077EC22510F5AA5A850204E36A9056083058B30E)
- fixes node url for coreum

### CODE REVIEW

File: modules/sdk-coin-coreum/test/unit/coreum.ts [hunk 1]
```diff
@@ -1,20 +1,24 @@
+import { CosmosTransaction, SendMessage } from '@bitgo/abstract-cosmos';
 import { BitGoAPI } from '@bitgo/sdk-api';
-import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
```
Comment (grsind19) [inline modules/sdk-coin-coreum/test/unit/coreum.ts:2]: @hitansh-bitgo  Are we not using it ?

**Review by grseetha — DISMISSED** [review]
LGTM

File: modules/sdk-coin-coreum/test/unit/coreum.ts (hunk 1 above)
Comment (hitansh-bitgo) [reply modules/sdk-coin-coreum/test/unit/coreum.ts:2]: We're still using it, the linter just sorted the imports

File: modules/sdk-coin-coreum/test/unit/coreum.ts [hunk 2]
```diff
@@ -388,4 +392,129 @@ describe('Coreum', function () {
...
+      sandBox.assert.calledOnce(tcoreum.getAccountBalance);
+      sandBox.assert.calledOnce(tcoreum.getAccountDetails);
+      sandBox.assert.calledOnce(tcoreum.getChainId);
+
+      const tcoreumTxn = new CosmosTransaction(coin, testnetUtils);
+      tcoreumTxn.enrichTransactionDetailsFromRawTransaction(res.serializedTx);
+      const tcoreumTxnJson = tcoreumTxn.toJson();
+      const sendMessage = tcoreumTxnJson.sendMessages[0].value as SendMessage;
+      const balance = new BigNumber(testBalance);
+      const actualBalance = balance.minus(new BigNumber(6250));
```
Comment (Vijay-Jagannathan) [inline modules/sdk-coin-coreum/test/unit/coreum.ts:443]: seems to be a magic number; where does it come from and can it be a constant ?

**Review by OttoAllmendinger — APPROVED** [review]

File: modules/sdk-coin-coreum/test/unit/coreum.ts (hunk 2 above)
Comment (hitansh-bitgo) [reply modules/sdk-coin-coreum/test/unit/coreum.ts]: updated

**Pull request merged**